SERVER/api/v1/controllers/adCtrl.js

```diff
@@ -137,11 +137,12 @@ const Ad = {
 
     // Seller update the status or price of the car/car ad.
 
-    async updateAd(req, res) {
+    async updateStatus(req, res) {
         const findAd = ads.find(ad => ad.car_id === parseInt(req.params.car_id));
         let m = moment();
```

Comment:
Hey, your variable names should be descriptive enough. Change `m` to a more meaningful name.